cmd/bios/bios.go

```diff
@@ -18,24 +19,38 @@ type biosActionFlags struct {
 	serverID string
 }
 
-func (f *biosActionFlags) ToCondition() (*types.ConditionCreate, error) {
-	id, err := f.ParseServerID()
+func CreateBiosControlCondition(ctx context.Context, action rctypes.BiosControlAction) error {
+	theApp := mctl.MustCreateApp(ctx)
+
+	client, err := app.NewConditionsClient(ctx, theApp.Config.Conditions, theApp.Reauth)
+	if err != nil {
+		return err
+	}
+
+	serverID, err := biosFlags.ParseServerID()
 	if err != nil {
-		return nil, err
+		return err
 	}
 
-	biosParams := rctypes.NewBiosControlTaskParameters(id, rctypes.ResetSettings)
+	params := rctypes.NewBiosControlTaskParameters(serverID, action)
 
-	params, err := json.Marshal(biosParams)
+	response, err := client.ServerBiosControl(ctx, params)
```

Comment:
are we burying parameters here?

Reply:
I'd prefer to get away from a pattern where we have a single `Kind` of condition that handles multiple things and thus any users of the API have to couple more tightly to our internal implementation details.

Reply:
I agree. I think that is a problem for another day though.

My reason is that, as we have talked about before, this would require some work in Ctrl to support. I can create a ticket to support this. I also think doing this would be a good first step for merging some controllers (biosCfg and Flipflop for example). We could purge the "Actions" at that time, and replace with multiple *Kinds*.